endpoints/openrtb2/auction.go

```diff
@@ -133,8 +123,15 @@ func (deps *endpointDeps) Auction(w http.ResponseWriter, r *http.Request, _ http
 	defer cancel()
 
 	usersyncs := usersync.ParsePBSCookieFromRequest(r, &(deps.cfg.HostCookie))
+	if req.Site != nil && req.Site.Publisher != nil {
+		labels.PubID = req.Site.Publisher.ID
+	}
 	if req.App != nil {
 		labels.Source = pbsmetrics.DemandApp
+		labels.RType = pbsmetrics.ReqTypeORTB2App
+		if req.App.Publisher != nil {
```

Comment:
Shouldn't we also check if `req.App.Publisher.ID != ""` ? 

Also, after this if condition is executed, do we need to check `labels.PubID = ""` and set it to `UNKNOWN` if it is?

Reply:
Yep, the code changed